packages/webview_flutter/webview_flutter_android/android/src/main/java/io/flutter/plugins/webviewflutter/WebViewClientHostApiImpl.java

```diff
@@ -113,6 +120,14 @@ public void onPageFinished(WebView view, String url) {
       flutterApi.onPageFinished(this, view, url, reply -> {});
     }
 
+    @Override
+    public void onReceivedHttpError(
+        WebView view, WebResourceRequest request, WebResourceResponse errorResponse) {
+      if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
```

Comment:
You can move this check above the method: `@RequiresApi(api = Build.VERSION_CODES.LOLLIPOP)`. See the method below as reference.